.github/workflows/pr-realm-js.yml

```diff
@@ -698,11 +698,32 @@ jobs:
             ~/.android/adb*
           key: avd-29
 
+      - name: Bundle test app
+        working-directory: integration-tests/environments/react-native-test-app
+        run: |
+          mkdir dist
+          npx react-native bundle --entry-file index.js --platform android --dev false --minify false --bundle-output dist/main.android.jsbundle --assets-dest dist/res
```

Comment:
We use `--minify false` to avoid test failures from error messages mismatching due to class names being renamed in the bundle.